### DISCUSSION TITLE
disabled props in useForm hook

### DISCUSSION THREAD

**Discussion by kaio-eduardo** [react-hook-form/react-hook-form#11738] [Q&A] [answered]
There is a disabled props in useForm hook that seems to work to set all the fields as disabled, but i did not find anything about it in the docs:``
```ts
const { control, watch, reset, formState, handleSubmit } = useForm({
    disabled: true,
  });
```
is it ok to use this option?

**Comment by sujinleeme** [comment, marked as answer]
Yes, the new `disabled` prop was introduced in the v7.48 version.
https://github.com/react-hook-form/react-hook-form/releases/tag/v7.48.01
